README.md

```diff
@@ -1,6 +1,6 @@
 # scaffolding-chef
 
-[![Build Status](https://buildkite.com/chef/chef-scaffolding-chef-master-habitat-build)]
+[![Build status](https://badge.buildkite.com/82171de46ba8cce19902553657cd1e5919400a9a52c5e4530e.svg)](https://buildkite.com/chef/chef-scaffolding-chef-master-habitat-build)
```

Comment:
```suggestion
[![Build status](https://badge.buildkite.com/82171de46ba8cce19902553657cd1e5919400a9a52c5e4530e.svg?branch=master)](https://buildkite.com/chef/chef-scaffolding-chef-master-habitat-build)
```

Reply:
I've filed an issue with BK already, but out of the box you don't get master with their badges

Reply:
You are a god